lib/output.js

```diff
@@ -1447,13 +1447,16 @@ function OutputStream(options) {
         }
     });
     DEFPRINT(AST_ObjectKeyVal, function(self, output){
-        var value;
+        var get_name = function() {
```

Comment:
More JIT friendly. Function generated only once. No captured variables - explicitly passed in.